staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -181,6 +181,22 @@ type VolumeSource struct {
 	//
 	// +optional
 	Ephemeral *EphemeralVolumeSource `json:"ephemeral,omitempty" protobuf:"bytes,29,opt,name=ephemeral"`
+	// image represents an OCI object (a container image or artifact) pulled and mounted on the kubelet's host machine.
+	// The volume is resolved at pod startup depending on which PullPolicy value is provided:
+	//
+	// - Always: the kubelet always attempts to pull the reference. Container creation will fail If the pull fails.
+	// - Never: the kubelet never pulls the reference and only uses a local image or artifact. Container creation will fail if the reference isn't present.
+	// - IfNotPresent: the kubelet pulls if the reference isn't already present on disk. Container creation will fail if the reference isn't present and the pull fails.
+	//
+	// The volume gets re-resolved if the pod gets recreated, which means that new remote content will become available on pod recreation.
```

Comment:
Since pods are mostly immutable, what does it mean to have a pod recreated?

Reply:
Good point, I "knew" that meant delete and recreated, but many users won't.

Reply:
Slightly modified the comment to include container deletion.